presentation/src/main/java/com/yapp/presentation/ui/member/qrcodescanner/QrCodeScanner.kt

```diff
@@ -250,4 +279,14 @@ fun InformText(modifier: Modifier = Modifier) {
             style = AttendanceTypography.body1
         )
     }
+}
+
+fun intentToAppSetting(context: Context) {
```

Comment:
```suggestion
private fun intentToAppSetting(context: Context) {
```
이 곳에서만 쓰이니 private 해주는게 어떨까용

Reply:
앗 요고 빼먹었네요 감사합니다!